README.md

```diff
@@ -34,78 +62,97 @@ re.test('👨🏾‍⚕️'); // '\u{1F468}\u{1F3FE}\u200D\u2695\uFE0F'
 // → true
 ```
 
-We propose to support the following Unicode sequence properties defined in [UTS18](https://unicode.org/reports/tr18/#Full_Properties) and [UTS51](https://unicode.org/reports/tr51/):
+We propose to support the following Unicode sequence properties defined in
+[UTS18](https://www.unicode.org/reports/tr18/#Full_Properties) and
+[UTS51](https://www.unicode.org/reports/tr51/):
 
 - `Basic_Emoji`
+- `Emoji_Keycap_Sequence`
 - `RGI_Emoji_Modifier_Sequence`
+- `RGI_Emoji_Flag_Sequence`
 - `RGI_Emoji_Tag_Sequence`
 - `RGI_Emoji_ZWJ_Sequence`
 - `RGI_Emoji`
 
 Each of these sequence properties expands to a finite, well-defined set of strings.
+(`Basic_Emoji` also applies to many single characters.)
 
-Over time, we can choose to support additional sequence properties, following the upstream Unicode Standard.
+Over time, we can choose to support additional properties of strings, following the upstream Unicode Standard.
 
 ## High-level API
 
 Re-using the existing Unicode property escapes syntax for this new functionality seems appropriate:
 
-<pre>\p{<b><i>UnicodeSequencePropertyName</i></b>}</pre>
+<pre>\p{<b><i>PropertyName</i></b>}</pre>
 
-The negated `\P{…}` form is not supported for sequence properties as it would be [a footgun](https://github.com/tc39/proposal-regexp-unicode-sequence-properties/issues/6#issuecomment-368460069). It’s not generally useful, and is better expressed as a negative lookahead. Compare the unsupported `/\P{UnicodeSequenceProperty}/u` (what should it do?) with `/(?!\p{UnicodeSequenceProperty})/u` (clear what it does).
+Where `PropertyName` can be one of the properties of strings listed above.
 
-Given that `UnicodeSequencePropertyName` expands to a list of sequences of Unicode code points, the proposal also includes a static restriction that bans such properties within character classes.
+The complement of such a property is not supported:
+Both `\P{PropertyName}` and `[^...\p{PropertyName}...]` will fail to compile (throw a `SyntaxError` exception)
+if `PropertyName` is a property of strings.
 
-### FAQ
-
-#### What about backwards compatibility?
+We have thought of possible definitions of such a complement, but we believe that they are not generally useful.
 
-Unicode property escapes for unsupported Unicode properties throw an early `SyntaxError`. As such, we can add support for new properties in a backwards-compatible way, as long as we re-use the existing syntax.
+Some of the use cases for “not a property of strings” can be supported via a negative lookahead:
+`/(?!\p{RGI_Emoji_Flag_Sequence})\p{Symbol}/u`
```

Comment:
Does ECMAScript support lookahead?

Reply:
Yes. The previous version of this readme already had this example, I just added the `\p{Symbol}` because it seems weird to have a lookahead with nothing following.

And if you look on stackoverflow for questions about how to do something like subtraction and intersection in JS regex, you will find workarounds involving lookaheads.